tests/sentry/integrations/slack/webhooks/actions/test_status.py

```diff
@@ -255,10 +255,12 @@ def test_archive_issue_until_escalating(self, mock_tags, mock_record):
         assert "via" not in blocks[4]["elements"][0]["text"]
         assert ":white_circle:" in blocks[0]["text"]["text"]
 
-        assert len(mock_record.mock_calls) == 2
-        start, halt = mock_record.mock_calls
-        assert start.args[0] == EventLifecycleOutcome.STARTED
-        assert halt.args[0] == EventLifecycleOutcome.SUCCESS
+        assert len(mock_record.mock_calls) == 4
+        start_1, success_1, start_2, success_2 = mock_record.mock_calls
+        assert start_1.args[0] == EventLifecycleOutcome.STARTED
+        assert success_1.args[0] == EventLifecycleOutcome.SUCCESS
+        assert start_2.args[0] == EventLifecycleOutcome.STARTED
+        assert success_2.args[0] == EventLifecycleOutcome.SUCCESS
```

Comment:
Do we have tests for the other dialog types? If so, could we also add metric assertions to those as well please?

Reply:
yep i am waiting for tests to fail, thinking there are other places that are missing